ton_client/client/src/contracts/run.rs

```diff
@@ -253,14 +284,43 @@ pub(crate) fn decode_unknown_output(_context: &mut ClientContext, params: Params
     let body = base64_decode(&params.bodyBase64)?;
     let result = Contract::decode_unknown_function_response_from_bytes_json(
         params.abi.to_string().to_owned(),
-        &body).map_err(|err|ApiError::contracts_decode_run_output_failed(err))?;
+        &body,
+        params.internal)
+            .map_err(|err|ApiError::contracts_decode_run_output_failed(err))?;
     Ok(ResultOfDecodeUnknownRun {
         function: result.function_name,
         output: serde_json::from_str(result.params.as_str())
             .map_err(|err| ApiError::contracts_decode_run_output_failed(err))?
     })
 }
 
+pub(crate) fn get_run_body(_context: &mut ClientContext, params: ParamsOfGetRunBody) -> ApiResult<ResultOfGetRunBody> {
```

Comment:
Function name is strange:) Can you call it like build_message_body or create_message_body or build_run_body?